### PR TITLE
feat: add sitemap plugin as recipe

### PR DESCRIPTION
**Pull request by Margauxfeslard**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: stefandoorn/sitemap-plugin/1.0/config/packages/sitemap_plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+imports:
+        - { resource: "@SitemapPlugin/Resources/config/config.yml" }
```
Comment (toofff) [inline stefandoorn/sitemap-plugin/1.0/config/packages/sitemap_plugin.yaml:2]: Attention, indent please (4 spaces)

File: stefandoorn/sitemap-plugin/1.0/config/routes/sitemap_plugin.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+        sylius_sitemap::
+        resource: "@SitemapPlugin/Resources/config/routing.yml"
```
Comment (toofff) [inline stefandoorn/sitemap-plugin/1.0/config/routes/sitemap_plugin.yaml:2]: Attention, indent please (4 spaces)

```
sylius_sitemap::
    resource: "@SitemapPlugin/Resources/config/routing.yml"
```

File: stefandoorn/sitemap-plugin/1.0/manifest.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "SitemapPlugin\\SitemapPlugin": ["all"]
```
Comment (toofff) [inline stefandoorn/sitemap-plugin/1.0/manifest.json:3]: `"SitemapPlugin\\SitemapPlugin": ["all" => true]`

**Review by toofff — APPROVED** [review]
After small fix

File: stefandoorn/sitemap-plugin/1.0/config/packages/sitemap_plugin.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+imports:
+    - { resource: "@SitemapPlugin/Resources/config/config.yml" }
+
```
Comment (dunglas) [inline stefandoorn/sitemap-plugin/1.0/config/packages/sitemap_plugin.yaml:3]: This blank line should be removed.

**Review by dunglas — CHANGES_REQUESTED** [review]

**Review by dunglas — APPROVED** [review]

**Pull request merged**